src/pages/Routes.tsx

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { Routes, Route } from "react-router-dom";
+
+import { useAppSelector } from "../hooks/useContext";
+
+import {
+  Welcome,
+  Auth,
+  Configuration,
+  Home,
+  NewWorkout,
+  ExerciseList,
+  Exercise,
+  NewProgram,
+  Creator,
+  WorkoutList,
+  Workout,
+  TrainerList,
+  Trainer,
+  ProgramList,
+  Program,
+  NotFound,
+} from "./index";
+
+const Routing: React.FC = () => {
+  const user = useAppSelector((state) => state.user);
+  let routes; // type?
+
+  if (user) {
+    if (!user.configured) {
```

Comment:
to na przyszłość,
do zarządzania rolami użyjemy tego https://casl.js.org/v4/en/package/casl-react